week1/homework/src/index.js

```diff
@@ -1,3 +1,73 @@
 'use strict';
 
-// Write the homework code in this file
+const http = require('http');
+
+let state = 10;
+const PORT = 8080;
+const server = http.createServer(handleRequest);
+
+server.listen(PORT, () => {
+  console.log(`Server started at: ${stateUrl}`);
+});
+
+function handleRequest(request, response) {
+  console.log('on request', request.url);
+  switch (request.url) {
+    case '/':
+      response.setHeader('Content-Type', 'text/html');
+      response.write('<h1>Main Page</h1>');
+      break;
+    case '/state':
+      response.setHeader('Content-Type', 'text/html');
+      response.write(`<h1>Current state of the application is: ${state}</h1>`);
+      break;
+    case '/add':
+      response.setHeader('Content-Type', 'text/html');
+      response.write(`<h1>1 is added to the current state: ${state + Number(1)}</h1>`);
+      state = state + 1;
```

Comment:
You can also change the `state` first and then write it to response so as not to duplicate the logic.

Also you don't need to wrap `1` in a `Number`.